samples/openapi3/client/petstore/dart-dio/petstore_client_lib_fake/lib/model/format_test.dart

```diff
@@ -49,7 +48,7 @@ abstract class FormatTest implements Built<FormatTest, FormatTestBuilder> {
 
     @nullable
     @BuiltValueField(wireName: r'binary')
-    Uint8List get binary;
+    MultipartFile get binary;
```

Comment:
This needs to stay `Uint8List`. The whole thing may be a little more complicated.

Reply:
Yeah I agree binary should stay Uint8List... wonder where this came from

Reply:
I just looked into this. Apparently there is no more `file` type in OAS3, so internally everything with `format: binary` is mapped to `file` now. 
See https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/ModelUtils.java#L659

Looks like we need to post-process either models or parameters to work around this. I think we should revert this to `UintList8`  and handle multipart parameters in `postProcessOperationsWithModels`. There is already an old block there which handles body params.

Reply:
```        typeMapping.put("file", "MultipartFile");
        typeMapping.put("binary", "Uint8List");
```

the binary is still Uint8List in the typeMapping - why does it become MultipartFile?

I'll have a deeper look when I have more time